content/tutorials/trainapp-fitbit.textile

```diff
@@ -0,0 +1,550 @@
+---
+title: Building a Clockface app for Fitbit using Ably
+section: tutorials
+index: 95
+---
+
+In this tutorial, we will see how to use the "Ably Realtime client library":/realtime to build a realtime London Tube schedule clockface app for Fitbit. Ably enables realtime data sharing using Pub/Sub messaging architecture via a concept called "channels":https://www.ably.io/channels.
```

Comment:
More a question than anything, are 'clockface apps' a well-known app type for Fitbits? If not, it'd be good to provide some context for what that is in the tutorial. If it's well known ignore me.

Reply:
It's a type of app that you can buy, so i believe it is a well known app type